CMS-Enterprise/mint-app
# [MINT-3170] BE Data Exchange Change History

# MINT-3170

## Description
1. Add auditing the table
2. Connect the shared Translation to the Mapping
3. Extend the type to satisfy the interface
4. Attach generic testing functionality for the new map



## How to test this change
1. Make a model plan,
2. Update Plan Data Exchange Approach
3. Run the translate command (can be selected from `mdv db:seed:interactive`
4. Either query the translated audit collection, or look in the database and verify that the data translated as expectd.


1.  

## PR Author Checklist




## PR Reviewer Guidelines

- It's best to pull the branch locally and test it, rather than just looking at the code online!
- When approving a PR, provide a reason _why_ you're approving it
  - e.g. "Approving because I tested it locally and all functionality works as expected"
  - e.g. "Approving because the change is simple and matches the Figma design"
- Don't be afraid to leave comments or ask questions, especially if you don't understand why something was done! (This is often a great time to suggest code comments or documentation updates)
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
